src/less-style/inputs.less

```diff
@@ -78,20 +78,22 @@
 }
 
 .fd-commander {
-    display: flex;
     background-color: @cc-bg;
-    padding: 4px 1px 3px 3px;
-    margin-right: 1px;
+    padding: 5px;
 
     input {
         border: 1px solid @cc-neutral-hi;
-        flex: 1;
+        height: 19px;
+        width: calc(~"100% - 15px");
```

Comment:
Yeah, `calc` is amazing. I want to say it was added to CSS sometime in the past few years and was available in LESS before that, but not 100% sure.